crates/solver/src/solver/score_computation.rs

```diff
@@ -51,7 +51,16 @@ impl ScoreCalculator {
             - self.gas_price_factor * gas_price / 10_000_000_000.
             - self.nmb_orders_factor * nmb_orders as f64;
         let success_probability = 1. / (1. + exponent.exp());
-        let score = success_probability * (surplus + fees - gas_amount * gas_price);
+        let obj = surplus + fees - gas_amount * gas_price;
+        let score: f64;
+        let cap = 0.01;
```

Comment:
Are you sure the scaling is correct? `gas_amount` and `gas_price` seem to be large numbers which suggests that wei are being used as unit. The cap should use the same scaling then.

Reply:
Pushed a fix for that as I think you are right.